test/fixtures/lightscript/named-arrow-functions/skinny-white-empty/expected.json

```diff
@@ -0,0 +1,123 @@
+{
+  "type": "File",
+  "start": 0,
+  "end": 9,
+  "loc": {
+    "start": {
+      "line": 1,
+      "column": 0
+    },
+    "end": {
+      "line": 2,
+      "column": 1
+    }
+  },
+  "program": {
+    "type": "Program",
+    "start": 0,
+    "end": 9,
+    "loc": {
+      "start": {
+        "line": 1,
+        "column": 0
+      },
+      "end": {
+        "line": 2,
+        "column": 1
+      }
+    },
+    "sourceType": "script",
+    "body": [
+      {
+        "type": "NamedArrowDeclaration",
```

Comment:
@rattrayalex 

Actually not as bad as I thought:
https://github.com/wcjohnson/babylon-lightscript/commit/997428e2a46ce0eb64dd2799cea0c2544fead504

Let me know if you want this plucked